.circleci/config.yml

```diff
@@ -0,0 +1,26 @@
+version: 2.1
+
+jobs:
+  unit_test:
+    machine:
+      enabled: true
+      image: "ubuntu-1604:201903-01"
```

Comment:
Do we need the machine executor here? Or would tests run faster in a Docker executor?

Reply:
Yes, so that I can run against py2 and py3.